python/lsst/atmospec/processStar.py

```diff
@@ -554,10 +554,10 @@ def makeResultPickleable(self, result):
         result.spectrum.disperser.theta = None
 
     def runAstrometry(self, butler, exp, icSrc):
-        refObjLoaderConfig = LoadIndexedReferenceObjectsTask.ConfigClass()
-        refObjLoaderConfig.ref_dataset_name = 'gaia_dr2_20191105'
+        refObjLoaderConfig = ReferenceObjectLoader.ConfigClass()
         refObjLoaderConfig.pixelMargin = 1000
-        refObjLoader = LoadIndexedReferenceObjectsTask(butler=butler, config=refObjLoaderConfig)
+        # TODO: needs to be an Input Connection
+        refObjLoader = ReferenceObjectLoader(config=refObjLoaderConfig)
```

Comment:
This change looks like it (further, kinda) breaks something that was already broken by DM-35671, so given that's all going to need repairing anyway, I don't see the harm in merging this now.